tests/test_minimal.py

```diff
@@ -41,6 +41,10 @@
 }
 
 
+@pytest.mark.xfail(
+    OS_VERSION == "15.6",
+    reason="known issue: rpm has too many dependencies in SLE 15.6",
```

Comment:
Looks like the faulty update got into 15.5 as well :-/

Reply:
yes.

but I have a fixup in the pipeline

Reply:
is it also a problem in the base image, so do we need to fix it in 15.4 ?